### DISCUSSION TITLE
How can I set an arbitrary value in a form?

### DISCUSSION THREAD

**Discussion by bzy107** [filamentphp/filament#9065] [Help] [answered]
For instance, 
if you want to include the updated user ID while keeping it hidden, the code below didn't work on the edit page:
```
Hidden::make('create_user_id')
    ->default(auth()->id())
```
The documentation also mentions that it's only valid on the creation page.
https://filamentphp.com/docs/3.x/forms/fields/getting-started#setting-a-default-value

**Comment by emmexx** [comment, marked as answer]
If you don't need the create_user_id in the form but you need to save it to the database, you can use [Customizing data before saving](https://filamentphp.com/docs/3.x/panels/resources/creating-records#customizing-data-before-saving) or [Customizing data before saving](https://filamentphp.com/docs/3.x/panels/resources/editing-records#customizing-data-before-saving).
You put one of those methods in the create or edit files of your resource.

**Reply by bzy107** [reply]
Thank you. I have resolved it.
